### PR TITLE
Update create pool

### PR DESCRIPTION
**Pull request by soring323**

### CODE REVIEW

File: modules/apps/101-interchain-swap/keeper/msg_server.go [hunk 1]
```diff
@@ -111,49 +121,55 @@ func (k Keeper) OnCreatePoolReceived(ctx sdk.Context, msg *types.MsgCreatePoolRe
...
 		return nil, types.ErrInvalidDenomPair
 	}
 
+	//TODO: Need to implement params module and market maker.
+	amm := types.NewInterchainMarketMaker(
+		pool,
+		types.DefaultMaxFeeRate,
+	)
+
+	lpToken, err := amm.DepositSingleAsset(*msg.Tokens[0])
```
Comment (liangping) [inline modules/apps/101-interchain-swap/keeper/msg_server.go:163]: Where do you send `msg.Token[0]` to escrowed address?

File: modules/apps/101-interchain-swap/keeper/msg_server.go [hunk 2]
```diff
@@ -18,20 +18,32 @@ func NewMsgServerImpl(keeper Keeper) types.MsgServer {
...
 		msg.SourcePort,
 		msg.SourceChannel,
 	)
 
+	// when add initial liquidity, we need to update pool token amount.
+	initialLiquidity := sdk.NewCoin(pool.PoolId, msg.Tokens[0].Amount)
+	pool.AddPoolSupply(initialLiquidity)
+
+	// mint lpToken.
+	err := k.MintTokens(ctx, sdk.MustAccAddressFromBech32(msg.Sender), initialLiquidity)
```
Comment (liangping) [inline modules/apps/101-interchain-swap/keeper/msg_server.go:40]: it would be risky since the counterparty deposit is not finished.

Comment (soring323) [reply modules/apps/101-interchain-swap/keeper/msg_server.go:40]: hmm, if counterparty deposit is not finished, that lp token has no meaning forever. so I think it's no risk. 
Only when deposit is finished in counter party side, at that time pool is ready for swap and lp token has meaning.

Comment (soring323) [reply modules/apps/101-interchain-swap/keeper/msg_server.go:40]: and then if counter party deposit is not finished and pool creator has no ability to treat that, at that time, he can withdraw his fund with that ip token from pool. 

of course at that time, we need to delete pool also, about that operation, need to improve.

**Pull request merged**